dags/pytorch_xla/configs/pytorchxla_torchbench_config.py

```diff
@@ -85,16 +85,16 @@ class R2_4(enum.Enum):
     TORCH_XLA_REPO_BRANCH = "-b v2.4.0-rc8"
 
   class R2_5(enum.Enum):
-    TORCH_XLA_TPU_WHEEL = "https://storage.googleapis.com/pytorch-xla-releases/wheels/tpuvm/torch_xla-2.5.0rc9-cp310-cp310-linux_x86_64.whl"
-    TORCH_XLA_CUDA_WHEEL = "https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.5.0rc9-cp310-cp310-linux_x86_64.whl"
+    TORCH_XLA_TPU_WHEEL = "https://storage.googleapis.com/pytorch-xla-releases/wheels/tpuvm/torch_xla-2.5.0rc10-cp310-cp310-linux_x86_64.whl"
+    TORCH_XLA_CUDA_WHEEL = "https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.5.0rc10-cp310-cp310-linux_x86_64.whl"
     TORCH = "torch==2.5.0"
     TORCHVISION = "torchvision==0.19.0"
```

Comment:
Note that the torchvision and torchaudio should be compatible with the torch. The final release test is not running in airflow at this time.

Reply:
thanks, do we know how to confirm the compatible torchvision and torchaudio now?

Reply:
torchvision looks like preparing 0.20 release too, let me confirm locally which torchvision does PyTorch 2.5.0 is using now with `pip3 install torch==2.5.0 torchvision --index-url https://download.pytorch.org/whl/test/cpu`

Reply:
ok, it installed torchvision 0.20:
```
(base) manfei@t1v-n-136f5173-w-0:~$ pip3 install torch==2.5.0 torchvision --index-url https://download.pytorch.org/whl/test/cpu
Looking in indexes: https://download.pytorch.org/whl/test/cpu
Collecting torch==2.5.0
  Downloading https://download.pytorch.org/whl/test/cpu/torch-2.5.0%2Bcpu-cp312-cp312-linux_x86_64.whl (174.6 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 174.6/174.6 MB 61.0 MB/s eta 0:00:00
Collecting torchvision
  Downloading https://download.pytorch.org/whl/test/cpu/torchvision-0.20.0%2Bcpu-cp312-cp312-linux_x86_64.whl (1.8 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.8/1.8 MB 24.9 MB/s eta 0:00:00
Collecting filelock (from torch==2.5.0)
  Downloading https://download.pytorch.org/whl/test/filelock-3.13.1-py3-none-any.whl (11 kB)
Collecting typing-extensions>=4.8.0 (from torch==2.5.0)
  Downloading https://download.pytorch.org/whl/test/typing_extensions-4.12.2-py3-none-any.whl (37 kB)
Collecting networkx (from torch==2.5.0)
```

Reply:
install torchaudio 2.5.0 too:
```
(base) manfei@xxx:~$ pip3 install torch==2.5.0 torchvision torchaudio --index-url https://download.pytorch.org/whl/test/cpu
Looking in indexes: https://download.pytorch.org/whl/test/cpu
Collecting torch==2.5.0
  Using cached https://download.pytorch.org/whl/test/cpu/torch-2.5.0%2Bcpu-cp312-cp312-linux_x86_64.whl (174.6 MB)
Collecting torchvision
  Using cached https://download.pytorch.org/whl/test/cpu/torchvision-0.20.0%2Bcpu-cp312-cp312-linux_x86_64.whl (1.8 MB)
Collecting torchaudio
  Downloading https://download.pytorch.org/whl/test/cpu/torchaudio-2.5.0%2Bcpu-cp312-cp312-linux_x86_64.whl (1.7 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.7/1.7 MB 20.2 MB/s eta 0:00:00
Requirement already satisfied: filelock in ./miniconda3/lib/python3.12/site-packages (from torch==2.5.0) (3.13.1)
```

Reply:
let's test with this compatible version now